settings.ini

```diff
@@ -1,7 +1,7 @@
 [versions]
-FIRMWARE_VERSION=v1.0.2
+FIRMWARE_VERSION=v1.0.2-1
 DIAGNOSTICS_VERSION=0739ab9
-CONFIG_VERSION=6d63852
+CONFIG_VERSION=5d3ad7a
```

Comment:
it is weird that this is showing as a diff....when this is already how it is in master

Reply:
Yeah. The base is one commit before but it should be diffing to HEAD. It gets confused today I guess, like we all did :)

Reply:
doesn't really matter - just weird...will dismiss